<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConnectionSettings</class>
 <widget class="QWidget" name="ConnectionSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1243</width>
    <height>168</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Connection settings</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="1" rowspan="2">
       <widget class="QWidget" name="widgetInputTCP" native="true">
        <layout class="QGridLayout" name="gridLayoutTCP">
         <item row="0" column="1">
          <widget class="QLabel" name="label_2">
           <property name="styleSheet">
            <string notr="true">font-size:8pt</string>
           </property>
           <property name="text">
            <string>Port</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="lineEditPort">
           <property name="maximumSize">
            <size>
             <width>130</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLineEdit" name="lineEditHostname"/>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="styleSheet">
            <string notr="true">font-size:8pt</string>
           </property>
           <property name="text">
            <string>Host address</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QRadioButton" name="radioButtonModbusTCP">
        <property name="text">
         <string>Modbus TCP</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="2" rowspan="2">
       <widget class="QWidget" name="widgetInputRTU" native="true">
        <layout class="QGridLayout" name="gridLayoutRTU">
         <item row="0" column="1">
          <widget class="QLabel" name="label_3">
           <property name="styleSheet">
            <string notr="true">font-size:8pt</string>
           </property>
           <property name="text">
            <string>Baud rate</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="comboBoxBaudRate"/>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="label_5">
           <property name="styleSheet">
            <string notr="true">font-size:8pt</string>
           </property>
           <property name="text">
            <string>Stop bits</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QLabel" name="label_6">
           <property name="styleSheet">
            <string notr="true">font-size:8pt</string>
           </property>
           <property name="text">
            <string>Parity</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QComboBox" name="comboBoxStopBits"/>
         </item>
         <item row="1" column="0">
          <widget class="QLineEdit" name="lineEditCOM"/>
         </item>
         <item row="1" column="3">
          <widget class="QComboBox" name="comboBoxParity"/>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_4">
           <property name="styleSheet">
            <string notr="true">font-size:8pt</string>
           </property>
           <property name="text">
            <string>Serial/COM port</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="radioButtonRTU">
        <property name="text">
         <string>Modbus RTU</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Connection status</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="1">
       <widget class="QLabel" name="labelConnectLED">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="checkBoxAutoReconnect">
        <property name="text">
         <string>Auto-Reconnect</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Preferred</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="labelConnected">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="pushButtonConnect">
        <property name="text">
         <string>Connect</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
